### PR TITLE
fixed bugs

### PR DESCRIPTION
**Pull request by KVTaniguchi**
The purpose of this PR is to fix some of the updates I made recently so that they actually work.
- fixed podfile for cocopods 1.0
- added highlight button text color
- corrected btn separator width
- corrected btn highlight background color

### CODE REVIEW

**Comment by ryang1428** [comment]
travis builds are failing.. I am thinking they aren't on cocoapods 1.0? I can install locally on 1.0

**Comment by KVTaniguchi** [comment]
I was afraid t hat would happen, I'll revert to 0.39 and push again like last time tonight or tomorrow morning

**Comment by ryang1428** [comment]
Still having issues here. You can specify which version of cocoapods to use in the travis.yml but it doesn't seem to be working.

File: Pod/Classes/URBNAlertView.m [hunk 1]
```diff
@@ -353,12 +353,14 @@ - (URBNAlertActionButton *)createAlertViewButtonWithAction:(URBNAlertAction *)ac
...
     btn.layer.borderWidth = self.alertStyler.buttonBorderWidth.floatValue;
     btn.contentEdgeInsets = self.alertStyler.buttonContentInsets;
+    
     btn.tag = index;
     btn.actionType = action.actionType;
     btn.alertStyler = self.alertStyler;
     
     [btn setTitle:action.title forState:UIControlStateNormal];
     [btn setTitleColor:titleColor forState:UIControlStateNormal];
+    [btn setTitleColor:self.alertStyler.buttonHighlightTitleColor forState:UIControlStateHighlighted];
```
Comment (ryang1428) [inline Pod/Classes/URBNAlertView.m]: will you ever need to have a different highlight color for cancel/destructive buttons? We currently have those styles in but if thats something you wont need then theres no need to get them in now

**Comment by ryang1428** [comment]
Builds are now passing 👍

**Comment by KVTaniguchi** [comment]
@ryang1428 added cancel and destructive button title high light colors

**Pull request merged**